### ISSUE TITLE
Line number 

### ISSUE THREAD

**Issue by astrekalova** [mp911de/logstash-gelf#42]
Hello,
I'd like to use logstash-gelf library to log with the gelf appender to a logstash agent. I am using logback library and I could not find the possibilty to include line numbers in gelf log messages. Is it possible and how can I configure it? Here is my GelfLogAppender configuration:
```
<appender name="GELF" class="biz.paluch.logging.gelf.logback.GelfLogbackAppender">
		<host>udp:localhost</host>
		<port>12201</port>
        <version>1.0</version>
		<extractStackTrace>true</extractStackTrace>
		<filterStackTrace>false</filterStackTrace>
		<mdcProfiling>true</mdcProfiling>
		<timestampPattern>yyyy-MM-dd HH:mm:ss,SSSS</timestampPattern>
		<includeFullMdc>true</includeFullMdc>
		<line>true</line>
	</appender>
```

**Comment by mp911de** [comment]
Hi @astrekalova 
this impl does not support it yet. `SourceClassName` and `SourceMethodName` are already provided. You can look up all the fields at http://logging.paluch.biz/default-fields.html

If you want to, you could send me over a PR for a new field like `SourceLineNumber`. All logging frameworks except java.util.logging can provide the line number.

**Comment by mp911de** [comment]
Added support for SourceLineNumber in log4j, log4j2, logback JBoss logging. Available in 1.7.0-SNAPSHOT, see https://oss.sonatype.org/content/repositories/snapshots/biz/paluch/logging/logstash-gelf/1.7.0-SNAPSHOT/

**Comment by mp911de** [comment]
Closing this issue, changes implemented.
